### PR TITLE
[Docs] Add Qwen3 forced alignment online example

### PR DESCRIPTION
**Pull request by taneem-ibrahim**
## Purpose
This PR adds an online `/pooling `example for` Qwen/Qwen3-ForcedAligner-0.6B`. Relevant issue: [#38805](https://github.com/vllm-project/vllm/issues/38805)

The motivation is that users can already run` Qwen3 ForcedAligner` through vLLM’s offline pooling API, but the online` /pooling` request shape is less obvious because it needs both audio input and reference text. This example gives users a simple ready to use script that shows the full request format, including messages, audio_url, task="token_classify", and the chat template needed to preserve the forced-alignment prompt.

This helps users who want to run forced alignment through a deployed online vLLM server instead of only through offline `LLM.encode(...).` .

## Test Plan
Start the vLLM server in online mode:

```
vllm serve Qwen/Qwen3-ForcedAligner-0.6B \
  --runner pooling \
  --enforce-eager \
  --trust-request-chat-template \
  --hf-overrides '{"architectures": ["Qwen3ASRForcedAlignerForTokenClassification"]}'
```
Once the startup is successful, in another terminal run:

`python examples/pooling/token_classify/forced_alignment_online.py`

## Test Result
```
Hello           0.000s - 0.160s
world           0.160s - 1.920s
```

This run uses the example’s default silent WAV input. The script output above shows the `/pooling` request accepts audio + reference text, returns token-classification logits, and decodes timestamp spans for each reference word.

### CODE REVIEW

**Review by yewentao256 — COMMENTED** [review]
Thanks for the work!

Could you copy-paste the full output with your script?

**Review by yewentao256 — COMMENTED** [review]
Also CC @wuchenhu98 if this solves your issue.

**Comment by taneem-ibrahim** [comment]
> Thanks for the work!
> 
> Could you copy-paste the full output with your script?

@wuchenhu98 added the online /pooling audio+text forced-alignment use case with output above in the PR body.

**Review by yewentao256 — APPROVED** [review]
I believe this closes https://github.com/vllm-project/vllm/issues/38805
Thanks for the work!
